### ISSUE TITLE
Intent to implement: Ooyala Player Embed

### ISSUE THREAD

**Issue by chibzu** [ampproject/amphtml#4320]
The goal is to implement a simple Ooyala element to allow embedding of videos from the Ooyala platform. It would follow an approach like:

```
<amp-ooyala
      height=400
      width=400
      data-embedcode="xkeHRiMjE6ls2aXoPoiqmPO6IU8HtXsg"
      data-pcode="5zb2wxOlZcNCe_HVT3a6cawW298X"
      data-playerid="26e2e3c1049c4e70ae08a242638b5c40">
    </amp-ooyala>
```

**Comment by jridgewell** [comment]
LGTM.

**Comment by chibzu** [comment]
I've created https://github.com/ampproject/amphtml/pull/4333 for the PR.

**Comment by rudygalfi** [comment]
This seems to have been closed by #4333.
